app/src/main/java/org/mozilla/fenix/wallpapers/WallpapersUseCases.kt

```diff
@@ -0,0 +1,246 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.wallpapers
+
+import android.content.Context
+import android.content.res.Configuration
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.os.StrictMode
+import androidx.annotation.VisibleForTesting
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import mozilla.components.concept.fetch.Client
+import mozilla.components.support.locale.LocaleManager
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.components.appstate.AppAction
+import org.mozilla.fenix.ext.settings
+import org.mozilla.fenix.perf.StrictModeManager
+import org.mozilla.fenix.utils.Settings
+import java.io.File
+import java.util.Date
+
+/**
+ * Contains use cases related to the wallpaper feature.
+ *
+ * @param context Used for various file and configuration checks.
+ * @param store Will receive dispatches of metadata updates like the currently selected wallpaper.
+ * @param client Handles downloading wallpapers and their metadata.
+ * @param strictMode Required for determining some device state like current locale and file paths.
+ *
+ * @property initialize Usecase for initializing wallpaper feature. Should be called eagerly.
```

Comment:
`Should be called eagerly.` This is somewhat ambiguous to me, as someone who has not yet used the Wallpapers API. What are you trying to convey here?

Reply:
I'll update this with some more detail, thanks for calling it out!

Reply:
Added some more detail